docs/development/manufacturer/config-target-guidance.md

```diff
@@ -0,0 +1,38 @@
+# Config Target Guidance - Cloud Build
```

Comment:
to make it explicitly the first one you can add this at the top of the file. But it needs to be a `.mdx` file afaik (not just `.md`)

```mdx
---
sidebar_position: 1
---
```

Reply:
confirmed.
mv to .mdx & add the above as line 1-3.
![image](https://github.com/betaflight/betaflight.com/assets/56646290/7205f592-4400-4d1b-badd-e66cad5d8f2e)

Reply:
Done.